packages/graphql-server/src/plugins/__tests__/__snapshots__/useRedwoodRealtime.test.ts.snap

```diff
@@ -54,7 +56,68 @@ exports[`useRedwoodRealtime should update schema with live directive 1`] = `
           },
           "description": undefined,
           "extensionASTNodes": [],
-          "extensions": {},
+          "extensions": {
+            Symbol(useRedwoodRealtime.wasLiveQueryAdded): true,
+          },
+        },
+      },
+    ],
+    [
```

Comment:
@dthyresson Are we happy with this containing a second entry?

Reply:
Good question. I confess I forgot to run tests and updates snapshots before committing. I’ll have to have another look. Thanks!

Reply:
Right. The snapshot test on the spy here isn't the most elegant. I think there is a better way now with tech extensions to see if the live directive was properly added

Reply:
Right. The snapshot test on the spy here isn't the most elegant. I think there is a better way now with tech extensions to see if the live directive was properly added

Reply:
@Josh-Walker-GM I made a new test that checks the schema before and after the `useRedwoodRealtime` plugin is applied -- and confirms the direct is added.

Reply:
I improved the test to check

* the original schema should not have the live directive before the useRedwoodRealtime plugin is applied
* and then the replaced schema should have the live directive afterwards